client/src/api/address-api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export default async function userLocation(latitude, longitude) {
+  const addressObj = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latitude},${longitude}&key=${process.env.REACT_APP_GOOGLE_API_KEY}`;
```

Comment:
i dont like how the queryparams are embedded in the string. see how we made api calls to our server and do it in the same format.